### PR TITLE
New package: pnpm.pnpm.7 version 7.9.0

### PR DESCRIPTION
**Pull request by russellbanks**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Sorry for the pnpm PR again; this is exactly the same as #68904, except this is for pnpm 7, because there are two channels that are currently getting updates: `pnpm 6` and `pnpm 7`, so it's best to split them now.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68910)

### CODE REVIEW

**Comment by russellbanks** [comment]
beep boop 1.2 schema

File: manifests/p/pnpm/pnpm/7/7.9.0/pnpm.pnpm.7.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+PublisherSupportUrl: https://github.com/pnpm/pnpm/issues
+# PrivacyUrl: 
+Author: pnpm contributors
+PackageName: pnpm 7
+PackageUrl: https://pnpm.io/
+License: MIT License
+LicenseUrl: https://github.com/pnpm/pnpm/blob/main/LICENSE
+Copyright: |-
+  Copyright (c) 2015-2016 Rico Sta. Cruz and other contributors
+  Copyright (c) 2016-2022 Zoltan Kochan and other contributors
```
Comment (Trenly) [inline manifests/p/pnpm/pnpm/7/7.9.0/pnpm.pnpm.7.locale.en-US.yaml:18]:
```suggestion
Copyright: Copyright (c) 2016-2022 Zoltan Kochan and other contributors
```
It's best to only include the active copyright, not the expired one

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Hitting virus scan issues;

**Pull request closed without merging**